internal/layer/sbom_restorer.go

```diff
@@ -29,10 +30,19 @@ type Cache interface {
 	RetrieveLayer(sha string) (io.ReadCloser, error)
 }
 
-func NewSBOMRestorer(layersDir string, logger Logger) SBOMRestorer {
+type SBOMRestorerOpts struct {
+	LayersDir string
+	Logger    Logger
+	Nop       bool
+}
```

Comment:
This struct makes it a bit clearer when we're calling the constructor what each option is for.